src/main/java/com/yahoo/tillyqb/rollcalulator/controllers/HelloController.java

```diff
@@ -0,0 +1,15 @@
+package com.yahoo.tillyqb.rollcalulator.controllers;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class HelloController
+{
+    @GetMapping("/hello/{name}")
+    public String getMoringGrettings(@PathVariable String name)
+    {
+        return "Hello " + name;
+    }
+}
```

Comment:
This controller can now be removed as we no longer need it for the example.

Reply:
This is now done